antismash/modules/pfam2go/__init__.py

```diff
@@ -31,9 +32,13 @@ def check_options(_options) -> List[str]:
 
 
 def check_prereqs() -> List[str]:
-    """Check for prerequisites"""
-    # Needs Pfam IDs, how to check?
-    return []
+    """Check for prerequisites
+        pfam2go-march-2018.txt: mapping file for Pfam to Gene Ontology mapping
+    """
+    failure_messages = []
+    if path.locate_file(path.get_full_path(__file__, 'data/pfam2go-march-2018.txt')) is None:
```

Comment:
You're breaking windows compatibility here, when you could use `path.get_full_path(__file__, 'data', 'pfam2go...`